spec/ndx-holographic-stimulation.extensions.yaml

```diff
@@ -22,24 +22,30 @@ groups:
     doc: link to the holographic stimulus pattern
     quantity: '?'
 - neurodata_type_def: HolographicSeries
-  neurodata_type_inc: OptogeneticSeries
+  neurodata_type_inc: TimeSeries
   doc: An extension of OptogeneticSeries to include the holographic representation
     for the stimulus.
+  attributes:
+  - name: unit
+    dtype: text
+    default_value: watts
+    doc: SI unit of data
+    required: false
   datasets:
   - name: data
-    neurodata_type_inc: VectorData
     dtype: float
     shape:
-    - null
-    - null
+    - - null
+    - - null
+      - null
```

Comment:
As discussed, this should simply be 2D at all times

Reference for RoiResponseSeries schema: https://github.com/NeurodataWithoutBorders/nwb-schema/blob/ec0a87979608c75a785d3a42f61e3366846ed3c2/core/nwb.ophys.yaml#L79-L81